Formula/kind.rb

```diff
@@ -1,8 +1,8 @@
 class Kind < Formula
   desc "Run local Kubernetes cluster in Docker"
   homepage "https://kind.sigs.k8s.io/"
-  url "https://github.com/kubernetes-sigs/kind/archive/v0.11.0.tar.gz"
-  sha256 "c78bed3a39afd2e523b38256b7f16445d0ad4b8b25f18b5766005e095af772cd"
+  url "https://github.com/kubernetes-sigs/kind/archive/refs/tags/v0.11.1.tar.gz"
```

Comment:
```suggestion
  url "https://github.com/kubernetes-sigs/kind/archive/v0.11.1.tar.gz"
```
We can use the shorter URL here.

Reply:
Done